compiler/src/codegen.rs

```diff
@@ -702,6 +728,22 @@ pub fn gen(
     results
 }
 
+#[cfg(feature = "protobufv3-codec")]
+pub fn gen(input_files: &[String]) -> anyhow::Result<()> {
+    let res = protobuf_codegen::Codegen::new()
```

Comment:
I don't get it. Isn't it just generating the protobuf files? The compiler of grpcio should only be used to generate service definition.

Reply:
I got confused, thinking it should generate both. Removed. Thanks